environment.yml

```diff
@@ -5,7 +5,7 @@ channels:
   - nodefaults
 
 dependencies:
-  - python>=3.9
+  - python>=3.10
```

Comment:
Should we fix this at the minimum or +1. I have recently run into issues where the latest version of python was installed and then ended up with datetime deprecation warnings. I ended up downgrading python to 3.11.

Just a thought.

Reply:
This would also be consistent with pixi.toml

Reply:
No strong sense for whether pixi.toml and environment.yml should be at min, max or in the middle. Looking elsewhere

- pandas is at [minimum](https://github.com/pandas-dev/pandas/blob/main/environment.yml#L6)
- numpy is [min + 1](https://github.com/numpy/numpy/blob/main/environment.yml#L10)

I don't know if this reflects different conventions for these projects, or if it is just haphazard.

I've hit the datetime warnings too. Maybe [this is the fix](https://til.simonwillison.net/python/utc-warning-fix)? I can give it a shot.